CHANGELOG.md

```diff
@@ -1,3 +1,259 @@
+
+##### old:
+
+```
+labels = "testing=true"
+
+taints = "nodeType=storage:NoSchedule"
+```
+
+##### new:
+
+```
+labels = {
+  "testing" = "true"
+}
+
+taints = {
+  "nodeType" = "storage:NoSchedule"
+}
+```
+
+#### Cluster upgrade steps
+
+Due to [Terraform](https://github.com/kinvolk/lokomotive/pull/824) and [Kubernetes](https://github.com/kinvolk/lokomotive/pull/1030) upgrades to v0.13+ and v1.19.3 respectively, there are some additional steps that you need to follow. Go to your cluster's directory and follow these steps:
+
+
+1. Upgrade local Terraform binary to version v0.13.X. You can follow the [following guide](https://learn.hashicorp.com/tutorials/terraform/install-cli.) to do that.
+
+2. Export your platform name and assets directory name used in your platform configuration. It will be used in next steps:
+  ```sh
+  export PLATFORM="packet" && export ASSETS_DIR="assets"
+  ``` 
+
+3. Remove old asset files:
+  ```sh
+  rm $ASSETS_DIR/terraform-modules/$PLATFORM/flatcar-linux/kubernetes/require.tf \
+  $ASSETS_DIR/terraform-modules/$PLATFORM/flatcar-linux/kubernetes/workers/require.tf \
+  $ASSETS_DIR/terraform-modules/dns/route53/require.tf 2> /dev/null
+  ```
+
+4. Go to terraform directory:
+  ```sh
+  cd $ASSETS_DIR/terraform
+  ```
+
+5. Replace old providers:
+  ```sh
+  terraform state replace-provider -auto-approve registry.terraform.io/-/ct registry.terraform.io/poseidon/ct && \
+  terraform state replace-provider -auto-approve registry.terraform.io/-/template registry.terraform.io/hashicorp/template
+  ```
+
+6. **NOTE:** delete old ReplicaSet `kube-apiserver` only for single node cluster.
+  ```sh
+  kubectl delete rs $(kubectl get rs -n kube-system -l k8s-app=kube-apiserver --no-headers=true  | sort -k5 -h | head -n +2 | awk '{print $1}')
+  ```
+
+7. FelixConfiguration to be managed by Helm:
+  ```sh
+  kubectl label FelixConfiguration default app.kubernetes.io/managed-by=Helm && \
+  kubectl annotate FelixConfiguration default meta.helm.sh/release-name=calico && \
+  kubectl annotate FelixConfiguration default meta.helm.sh/release-namespace=kube-system
+  ```
+
+Finally, run:
+
+```sh
+lokoctl cluster apply --skip-components -v
+```
+
+The update process typically takes about 10 minutes.
+After the update, running `lokoctl health` should result in an output similar to the following:
+
+```sh
+Node                     Ready    Reason          Message
+
+lokomotive-controller-0  True     KubeletReady    kubelet is posting ready status
+lokomotive-1-worker-0    True     KubeletReady    kubelet is posting ready status
+lokomotive-1-worker-1    True     KubeletReady    kubelet is posting ready status
+lokomotive-1-worker-2    True     KubeletReady    kubelet is posting ready status
+Name      Status    Message              Error
+
+etcd-0    True      {"health":"true"}
+```
+
+#### Cluster nodes component upgrade (optional)
```

Comment:
- How about "Upgrading native kubelets and etcd"? I find "cluster nodes component" very ambiguous.
- What does "optional" mean? As a user, how do I know whether I should do this?